docs/topics/proc_accessing-jolokia-using-hawtio-on-openshift.adoc

```diff
@@ -0,0 +1,32 @@
+[#accessing-jvm-metrics-using-jolokia]
+= Accessing JVM metrics using Jolokia on OpenShift
+
+link:https://jolokia.org/documentation.html[Jolokia^] is a built-in lightweight solution for accessing JMX (Java Management Extension) metrics over HTTP on OpenShift.
+It allows you to access CPU, storage, and memory usage data collected by JMX over an HTTP bridge using a REST interface and JSON-formatted message payloads, making it suitable for monitoring cloud applications, due to its comparably high speed and low resource-requirements.
+
+
+All Fabric8 container images expose port `8778` by default on the host of your {OpenShiftLocal}.
+
+[accessing-jvm-metrics-using-hawtio-console]
+= Accessing Jolokia metrics using the hawt.io console
+
+For Java-based applications, the OpenShift Web console provides the integrated link:https://docs.openshift.com/container-platform/3.6/architecture/infrastructure_components/web_console.html#jvm-console[_hawt.io_ console^] that collects and displays all relevant metrics output by the JVM running your application.
+
+.Prerequistes
+
+* the `oc` client uthenticated
+* a Java-based application container running in a project on OpenShift
+* port `8778` exposed in the container image (exposed by default in Fabric8 application images)
+* latest link:https://github.com/jboss-container-images/redhat-openjdk-18-openshift-image/blob/openjdk18-dev/image.yaml[JDK 1.8.0 image^]
```

Comment:
Maybe don't mention the Fabric8 images at all and instead explain how to enable it on _this_ image?